framework/src/modules/chain/bft/finality_manager.js

```diff
@@ -304,22 +304,39 @@ class FinalityManager extends EventEmitter {
 			return true;
 		}
 
+		// Order the two block headers such that earlierBlock must be forged first
+		let earlierBlock = delegateLastBlock;
+		let laterBlock = blockHeader;
```

Comment:
later can be future tense and the block is not saved, so why not present tense? something like `currentBlock`?